helm/prometheus-operator-app/values.yaml

```diff
@@ -8,6 +8,10 @@ ciliumNetworkPolicy:
   enabled: false
 
 prometheus-operator-app:
+  global:
+    rbac:
+      pspAnnotations:
+        seccomp.security.alpha.kubernetes.io/allowedProfileNames: '*'
```

Comment:
Can you check that this works also for ksm, node-exporter and grafana?

Reply:
> Can you check that this works also for ksm, node-exporter and grafana?

...and alertmanager.
Yep, works :+1:

Reply:
Nice :)